x-pack/platform/packages/shared/kbn-ai-assistant/src/utils/get_model_options_for_inference_endpoints.test.ts

```diff
@@ -14,27 +15,82 @@ import {
   ModelOptionsData,
 } from './get_model_options_for_inference_endpoints';
 import type { InferenceAPIConfigResponse } from '@kbn/ml-trained-models-utils';
+import {
+  DEFAULT_ELSER_INFERENCE_ID,
+  E5_SMALL_INFERENCE_ID,
+  ELSER_IN_EIS_INFERENCE_ID,
+} from '@kbn/observability-ai-assistant-plugin/public';
+
+const samplePreConfiguredInferenceEndpoints = [
+  {
+    inference_id: DEFAULT_ELSER_INFERENCE_ID,
+    task_type: 'sparse_embedding' as InferenceTaskType,
+    service: 'elasticsearch',
+    service_settings: {
+      num_threads: 1,
+      model_id: '.elser_model_2_linux-x86_64',
+      adaptive_allocations: {
+        enabled: true,
+        min_number_of_allocations: 0,
+        max_number_of_allocations: 32,
+      },
+    },
+  },
+  {
+    inference_id: E5_SMALL_INFERENCE_ID,
+    task_type: 'text_embedding' as InferenceTaskType,
+    service: 'elasticsearch',
+    service_settings: {
+      num_threads: 1,
+      model_id: '.multilingual-e5-small_linux-x86_64',
+      adaptive_allocations: {
+        enabled: true,
+        min_number_of_allocations: 0,
+        max_number_of_allocations: 32,
+      },
+    },
+  },
```

Comment:
Right now these tests are kinda hard to read because of all the "noise". I think we only care about `inference_id`

```suggestion
const samplePreConfiguredInferenceEndpoints = [
  {
    inference_id: DEFAULT_ELSER_INFERENCE_ID,
  },
  {
    inference_id: E5_SMALL_INFERENCE_ID,
  },
```

Perhaps just get rid of `samplePreConfiguredInferenceEndpoints`, and then have a couple of test cases that test different scenarios